modules/storage-api/src/test/java/org/apache/ignite/internal/storage/basic/TestMvPartitionStorageTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.storage.basic;
+
+import java.util.List;
+import org.apache.ignite.internal.storage.AbstractMvPartitionStorageTest;
+import org.apache.ignite.internal.storage.MvPartitionStorage;
+import org.junit.jupiter.api.BeforeEach;
+
+/**
+ * MV partition storage test implementation for {@link TestMvPartitionStorage} class.
+ */
+public class TestMvPartitionStorageTest extends AbstractMvPartitionStorageTest {
+    /** Test partition storage instance. */
+    private TestMvPartitionStorage storage;
+
+    @BeforeEach
+    void setUp() {
+        storage = new TestMvPartitionStorage(List.of());
```

Comment:
The initialization of `storage` can be moved to the field initializer, making the field `final` and also making the `@BeforeEach` unnecessary (so the code as a whole simpler).

Reply:
Ok, why not